src/StaticFAQ/FAQExtraInfo/BaggageInfo/BaggageDescription.js

```diff
@@ -89,11 +105,16 @@ export const BaggageDescription = ({
 export default createFragmentContainer(
   BaggageDescription,
   graphql`
-    fragment BaggageDescription on Baggage {
-      height
-      weight
-      width
-      length
+    fragment BaggageDescription on BookingBaggage {
```

Comment:
I would prefer to define prop name with `_propName` since we use it elsewhere

Reply:
What do you mean? Sorry, I don't understand this one.

Reply:
`BaggageDescription` => `BaggageDescription_baggage`

Reply:
`GraphQLCompilerContext: Unknown document `BaggageDescription`.`